tests/main_test.py

```diff
@@ -97,8 +103,8 @@ def test_bad_payment(self):
                 'preferred': 'cashapp'
             }
```

Comment:
add test cases for overly long values for each (make sure the API has some kind of max length as well)
Also, perhaps we shouldn't allow spaces in the username either.